src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -427,7 +427,7 @@ public bool Equals(Symbol member1, Symbol member2)
                 ArrayBuilder<TypeParameterSymbol> builder = ArrayBuilder<TypeParameterSymbol>.GetInstance(arity);
                 var typeParameters2 = member2.GetMemberTypeParameters();
 
-                for (int i = arity - 1; i >= 0; i--)
+                for (int i = 0; i < arity; i++)
```

Comment:
@AlekseyTs 

This was iterating through the type parameters in reverse order, and as a result caused https://github.com/dotnet/roslyn/issues/34583

It appears that you wrote this line in https://github.com/dotnet/roslyn/commit/5afa1df19994198694fed709ac20d92fd3cef878.

Is there any reason that the reverse order was used?

Thanks.

Reply:
Based on the LDM decision in dotnet/csharplang#2378 (comment) this entire section of code can be deleted, so this is no longer relevant.